src/CheckersBoard.java

```diff
@@ -505,14 +508,62 @@ public String toString(){//prints out the checkerBoard
 			}else if(i < 32){
 				result += " " + pieces.get(i) + " |   |";
 			}
-				
+
 			if(i == 3 || i == 11 || i == 19 || i == 27){
 				result += "|";
 			}
 		}//end for
 		result += "\n-+---+---+---+---+---+---+---+---+\n";
 		return result;
+
+
 	}//end toString()
+
+	public String cleanCoord(String s){
```

Comment:
Code Climate is mad about the wall of `if`s...  Bonus 25 XP for rewriting it to use simpler conditions.  And another bonus 25 XP if it correctly handles "invalid" board coordinates like B2 (on the board, but not used for checkers).

Reply:
This is way nicer!

Reply:
And this gets the full 50 bonus XP (this won't break, and rejecting invalid coordinates is currently handled by `findCoordinates()`.